andela/ah-frontend-space
# #166880071 setup sass

#### What does this PR do?
- allows using of sass with WebPack 

#### Description of Task to be completed?
- add the necessary loaders to compile the files with extensions .css and .scss to regular CSS.

#### How should this be manually tested?
- Clone this repo and run ``` npm install``` and then ``` npm start```.
- You will see that the styling in the .scss file is picked up by the template and that the word color is purple.

<img width="415" alt="Screenshot 2019-07-03 at 17 17 30" src="https://user-images.githubusercontent.com/43015966/60599067-7855ac00-9db6-11e9-98b2-ed0a6d8580bd.png">

#### What are the relevant pivotal tracker stories?
[#166880071](https://www.pivotaltracker.com/story/show/166880071)
